tests/resources/Vsphere-Util.robot

```diff
@@ -234,3 +234,14 @@ Check VM Guestinfo
     ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info -e ${vm} | grep ${str}
     Should Be Equal As Integers  ${rc}  0
     [Return]  ${output}
+
+Clean VMOMI Sessions
+    [Arguments]  ${user-agent}
+    Run  for x in $(govc session.ls); grep "${user-agent}" | xargs govc session.rm
```

Comment:
If that's the case my other check won't work either and random tests will fail if other tests are running at the same time as this check occurs. I didn't realize we were running tests in parallel against the same hosts. This test may not be possible to implement in the infrastructure as I now understand it.